packages/ant-stack/src/cli/commands/create.ts

```diff
@@ -1,24 +1,69 @@
-import { type } from "arktype";
 import chalk from "chalk";
 import { consola } from "consola";
+import { existsSync, mkdirSync, writeFileSync } from "fs";
+import { join } from "path";
+import { format } from "prettier";
+
+import { getConfig } from "../../config.js";
+
+const packageJson = (name: string) =>
+  `{"name": "api-${name
+    .slice(1)
+    .replace(
+      /\//g,
+      "-"
+    )}","version": "0.0.0","type": "module","scripts":{"dev": "ant dev","build": "ant build","start": "node dist"},"dependencies": {"ant-stack": "*"},"devDependencies":{"@types/node": "18","tsx": "*","typescript": "~5.0"}}`;
+
+const imports = `import { createErrorResult, createOKResult, type InternalHandler, zeroUUID } from "ant-stack"\n\n`;
+
+const handlers = (method: string) =>
+  `export const ${method}: InternalHandler = async (event) => {return createOKResult({}, zeroUUID);};`;
 
 export async function interactiveCreate() {
-  consola.info(chalk.blue("Create a new endpoint interactively!"));
-  await consola.prompt("What is the name of the endpoint?", { type: "text" });
-  await consola.prompt("What HTTP methods does it use?", {
+  const config = await getConfig();
+  consola.info(chalk("Creating a new endpoint."));
+  let path = "",
+    success = false;
```

Comment:
```suggestion
  let path = "",
  let success = false;
```
I don't really like multiple assignments on the same line because it's weird to my smol brain :pensive: